test_contract/lib.rs

```diff
@@ -19,6 +19,20 @@ mod test_contract {
         newtype1_val: NewType1,
     }
 
+    /// Example docs for an event.
+    /// They are multiline.
+    #[ink(event)]
+    pub struct Event1 {
+        /// Example docs for an event field.
+        /// They are multiline.
+        #[ink(topic)]
+        a: u32,
+        b: Struct2,
```

Comment:
I'd maybe add a struct/event that share the same name to showcase why your solution, with putting events into a separate `mod event`, is necessary. Would work as a test if someone tries to remove the `mod event { ... }` in the future.